### DISCUSSION TITLE
Jamulus-as-a-service

### DISCUSSION THREAD

**Discussion by pcar75** [jamulussoftware/jamulus#1174] [General]
Bonjour,
I was wondering if this was OK with the developers / maintainers.
I stumbled upon a kind of PaaS (Platform-as-a-Service) that dub themselves Jamulus-as-a-Service (google it).
In Montréal, our drummer and I tried their free 1-hour private server yesterday afternoon and had very good latency  and audio quality similar to a very good local public server here called "Psilly". 
While I don't want to advertise a commercial site (pay by the hour if more than one) , I think it's a great idea.
Any thoughts ...

**Comment by gene96817** [comment]
If there is a shortage of free public servers, then a paid service could succeed. 
I hope we build a community that shares servers. 
I am concerned that there seems to be a growing attitude that "you can do whatever you want by just installing a private server".

**Comment by dcorson-ticino-com** [comment]
There are several other such services.
With the number of unused public servers out there that one could use, I don't know if is really needed.
But if they can offer better service, lower ping times, etc., why not?

**Comment by MITEL-UNIUD** [comment]
There are at least other 3 well-known services in the community, one of which is also currently free (Melomax, Koord, MusicBridge - in order of creation). I do no think these services compete with free unused servers: public servers were for the pre-Covid Jamulus use case (jamming), while now a lot of people need private rehearsal places. This is possible due to the kind of open source licensing, and also somewhat good for Jamulus, because the step is lowered for those bands/choirs without a computer nerd inside :)

**Reply by gene96817** [reply]
I'll never speak against an entrepreneur delivering value. If I recommend a commercial serv, then I need a method to create a budget for a group. (Plus the server needs to be within 1000 miles of Honolulu. Closer would be better.  )

**Comment by pcar75** [comment]
I gave some feedback to a provider (Melomax) and they were really helpfull :-)))
